pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -10647,6 +11110,9 @@ def test_phase_dict_helper_errs(sma_mir, arg_dict, dummy_phase_dict, msg):
     assert str(cm.value).startswith(msg)
 
 
+# TODO should this be removed? select_mask wasn't used in phase_dict_helper so I
+# removed it. But I don't really understand what this test is doing so not sure if it
+# should just be removed too.
```

Comment:
@kartographer maybe you can chime in here?

Reply:
I think this was a leftover after trying to chase down a problem related to using the selection screen in `phase`.  I think it can be deleted without issue.